external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/v2/V2JDBCTest.scala

```diff
@@ -180,5 +180,14 @@ private[v2] trait V2JDBCTest extends SharedSparkSession with DockerIntegrationFu
       testCreateTableWithProperty(s"$catalogName.new_table")
     }
   }
+
+  def testIndex(tbl: String): Unit = {}
```

Comment:
Did you want to add some test code here? If this is empty, the following test case is invalid.

Reply:
I intentionally leave it empty and let the subclass (e.g. `MySQLIntegrationSuite`) implement it.

Reply:
I understand that we leave it empty because other dialects do not support index yet. how about
```
def supportIndex: Boolean = false

test(...) {
  require(supportsIndex)
  test code
}
```